app/finders/songs_finder.rb

```diff
@@ -1,26 +1,26 @@
 class SongsFinder
-  
+
   def self.search(keywords)
     return [] if keywords.blank?
-    weighted_query = ActiveRecord::Base::sanitize(">(#{keywords})")
-    unweighted_query = ActiveRecord::Base::sanitize(keywords)
+    weighted_query = ActiveRecord::Base::sanitize(">(#{keywords}*)")
```

Comment:
Excellent, I was thinking that only the last term should be partial match. I think you'll also have to strip the space at the end, lest you end up with something like this: `term1 term2 *`.